modules/reindex/src/main/java/org/elasticsearch/index/reindex/BulkIndexByScrollResponse.java

```diff
@@ -155,19 +146,9 @@ public void readFrom(StreamInput in) throws IOException {
         super.readFrom(in);
         took = TimeValue.readTimeValue(in);
         status = new BulkByScrollTask.Status(in);
-        int indexingFailuresCount = in.readVInt();
-        List<Failure> indexingFailures = new ArrayList<>(indexingFailuresCount);
-        for (int i = 0; i < indexingFailuresCount; i++) {
-            indexingFailures.add(new Failure(in));
-        }
-        this.indexingFailures = unmodifiableList(indexingFailures);
-        int searchFailuresCount = in.readVInt();
-        List<ShardSearchFailure> searchFailures = new ArrayList<>(searchFailuresCount);
-        for (int i = 0; i < searchFailuresCount; i++) {
-            searchFailures.add(readShardSearchFailure(in));
-        }
-        this.searchFailures = unmodifiableList(searchFailures);
-        this.timedOut = in.readBoolean();
+        indexingFailures = in.readList(Failure::new);
```

Comment:
Should we rename this to `bulkFailures` to make it more explicit that it happened during the bulk phase?